### ISSUE TITLE
texto 5789

### ISSUE THREAD

**Issue by arademaker** [cpdoc/dhbb#56]
> COMPANHIA VALE DO RIO DOCE (VALE) : A composição acionária da Valepar sofreu mais duas importantes alterações. Em novembro de 2003, a BNDESPAR adquiriu 8,5% do capital da «holding», pagando R$1,5 bilhão pelas ações que pertenciam o clube de investimentos do empregados da companhia. Dirigido na época pelo economista Carlos Lessa, o banco informou que o negócio obedeceu a interesses estratégicos, manifestando preocupação com o risco de controle da Vale por grupos internacionais. Em março do ano seguinte, a Bradespar adquiriu 99% da participação que o banco Opportunity detinha na Elétron. A Bradespar e o Opportunity segregaram suas participações acionárias diretas e indiretas na Valepar, antes detidas diretamente por meio da Elétron. 

Que banco?

**Comment by vcvpaiva** [comment]
O banco "Opportunity"?

**Comment by suemi-higuchi** [comment]
Eu entendo que seria o BNDESPAR, que é o Banco Nacional de Desenvolvimento Participações S.A.

**Comment by arademaker** [comment]
Sim, também prefiro esta interpretação, mas então temos que mudar o artigo em `a BNDESPAR adquiriu 8,5%...` ... porque isto é que está causando confusão. Em tempo, a sigla BNDESPAR não foi introduzida no verbete.  Imagino que vc obteve esta informação do verbete `5846.text` que diz `BNDES Participações (BNDESPAR)`.
